sample/src/commonMain/kotlin/com/swmansion/kmpsharing/sample/App.kt

```diff
@@ -56,7 +61,35 @@ fun App() {
                     shape = RoundedCornerShape(12.dp),
                 ) {
                     Text(
-                        text = "Share a file!",
+                        text = "Share a link!",
+                        style = MaterialTheme.typography.titleMedium,
+                        fontWeight = FontWeight.Medium,
+                    )
+                }
+                Spacer(modifier = Modifier.height(16.dp))
+                Button(
+                    onClick = {
+                        bitmap?.let {
+                            share(
+                                data = it,
+                                options =
+                                    SharingOptions(
+                                        android = AndroidSharingOptions(dialogTitle = "Test Image!")
```

Comment:
Does the title show up anywhere? When I tested this, I saw the default title, which led me to believe this was ignored for image sharing. If that is the case, let’s remove it.